### PR TITLE
Fix missing Spanish translations

### PR DESCRIPTION
**Pull request by CodingAleCR**
## Changelog:
- Translated missing keys: saintBeatriceOfSilvaVirgin, saintBlaseBishopAndMartyrSaintAnsgarBishop, saintMariaDeJesusSacramentadoVenegasVirgin, saintTeresaOfCalcuttaReligious.
- Removed key saintBeunoAbbot as this is not a celebration that Spanish speaking countries keep. As a side note, I could not find a reliable translation for this one.

### CODE REVIEW

**Comment by tukusejssirs** [comment]
On `Saint Beuno`: not a reliable source, but I could find `San Beuno` or `San Beuno de Gales` ([src: Pinterest](https://www.pinterest.es/pin/56928382770025423/)). Even in [Portuguese](https://pt.wikipedia.org/wiki/S%C3%A3o_Beuno), they use the same name.

On a side note: as I have noted before, you should not _remove_ any key of celebrations not celebrated in Spanish speaking countries; I even encouraged to add them all, but that is not an obligatory condition in order merge it into romcal.

**Comment by tukusejssirs** [comment]
And you still missed [`saintAlbanMartyr`](https://github.com/romcal/romcal/blob/8f84411cebb65a47bf385d3fe72292a966227272/src/locales/es.ts#L142).

**Comment by tukusejssirs** [comment]
On `saintJeanVianneyTheCureOfArsPriest`: Current value is `San Juan Vianney (el Cura de Ars), Sacerdote`. Do the liturgical books use the `(el Cura de Ars)`? I have already removed it from romcal in `en.ts`.

File: src/locales/es.ts [hunk 1]
```diff
@@ -238,11 +237,11 @@ export default {
     saintJuanDiego: 'San Juan Diego',
     saintJuanDiegoCuauhtlatoatzin: 'San Juan Diego Cuauhtlatoatzin',
     saintLeoTheGreatPopeAndDoctor: 'San Leo el Magno, Papa y Doctor',
-    saintLorenzoRuizAndCompanionsMartyrs: 'Saint Lorenzo Ruiz y Companions, Mártires',
+    saintLorenzoRuizAndCompanionsMartyrs: 'San Lorenzo Ruiz y Compañeros, Mártires',
     saintLouisSaintJosephOfCalasanzPriest: 'San Luis/San José de Calasanz, Sacerdote',
     saintMargaretMaryAlacoque: 'Santa Margarita María Alacoque',
```
Comment (tukusejssirs) [inline src/locales/es.ts]: Should it be `Santa Margarita María Alacoque` or `Santa Margarita María de Alacoque`? There are two (duplicate) keys of her celebration and you have localised it two different ways.

**Review by tukusejssirs — CHANGES_REQUESTED** [review]

File: src/locales/es.ts (hunk 1 above)
Comment (CodingAleCR) [reply src/locales/es.ts]: Both are accepted although I think the most accurate would be "de Alacoque", will change that.

**Comment by CodingAleCR** [comment]
> On `saintJeanVianneyTheCureOfArsPriest`: Current value is `San Juan Vianney (el Cura de Ars), Sacerdote`. Do the liturgical books use the `(el Cura de Ars)`? I have already removed it from romcal in `en.ts`.

You are right, will remove that.

**Comment by CodingAleCR** [comment]
As a side note, I´ve found this, that might be helpful for you guys, I´ve long used it for Spanish but just found out that it is also available in English. It´s a page called [Santopedia](http://en.santopedia.com/) that compiles quite a big list of saints, and their celebration dates.

**Comment by CodingAleCR** [comment]
> On `Saint Beuno`: not a reliable source, but I could find `San Beuno` or `San Beuno de Gales` ([src: Pinterest](https://www.pinterest.es/pin/56928382770025423/)). Even in [Portuguese](https://pt.wikipedia.org/wiki/S%C3%A3o_Beuno), they use the same name.
> 
> On a side note: as I have noted before, you should not _remove_ any key of celebrations not celebrated in Spanish speaking countries; I even encouraged to add them all, but that is not an obligatory condition in order merge it into romcal.

I'll add it back and use the same name, then, just translate the other words around it.

**Comment by tukusejssirs** [comment]
> As a side note, I´ve found this, that might be helpful for you guys, I´ve long used it for Spanish but just found out that it is also available in English. It´s a page called [Santopedia](http://en.santopedia.com/) that compiles quite a big list of saints, and their celebration dates.

I know it, but I don’t regard it as an official source. From my point of view, we should try to use _Martyrologium Romanum_ from 2004 as the base of the romcal’s own martyrology library. Then we might need to add some of other saints/blesseds that were removed from the MR2004, but are celebrated in one or more particular calendars.

That said I use that website. :smiley:

**Review by tukusejssirs — APPROVED** [review]

**Comment by tukusejssirs** [comment]
@CodingAleCR, you wanted to localise keys of all celebrations used in any Spanish speaking country, but you have missed many. Note that most of the calendars listed below does not contain all the celebration that they should contain (you might want to fix that too).

I have created a coverage list of all keys localised to Spanish while checking only the Spanish speaking countries. The list of Spanish speaking coutries is acquired from [here](https://www.nationsonline.org/oneworld/countries_by_languages.htm#Spanish).

Explanation of the table header:
- `Calendar` is the calendar name in romcal;
- `Locale` is the locale name in romcal (in this table, only `es`);
- `Coverage` means how many keys used in that particular calendar are localised into the particular locale with the percentage in the brackets;
- `Total keys in calendar` is the number of keys used in the calendar.

| Calendar          | Locale | Coverage    | Total keys in calendar |
|-------------------|--------|-------------|------------------------|
| argentina         | es     | 12 (57 %)   | 21                     |
| bolivia           | es     | 9 (52 %)    | 17                     |
| chile             | es     | 11 (84 %)   | 13                     |
| costaRica         | es     | 1 (50 %)    | 2                      |
| guatemala         | es     | 1 (100 %)   | 1                      |
| mexico            | es     | 12 (54 %)   | 22                     |
| panama            | es     | 1 (100 %)   | 1                      |
| paraguay          | es     | 1 (100 %)   | 1                      |
| peru              | es     | 9 (75 %)    | 12                     |
| philippines       | es     | 5 (50 %)    | 10                     |
| puertoRico        | es     | 9 (75 %)    | 12                     |
| spain             | es     | 16 (50 %)   | 32                     |
| unitedStates      | es     | 3 (15 %)    | 20                     |
| uruguay           | es     | 1 (100 %)   | 1                      |
| venezuela         | es     | 1 (100 %)   | 1                      |

---

@emagnier, I have squashed the changes into #213, however, if you want, you can merge this PR.

**Comment by CodingAleCR** [comment]
I mostly wanted to add the base language for this, I think for each country it might be a good idea too look for contributors so that they can also add national holidays or related Saints. I did plan to do Costa Rica from a separate PR. 

Also, there's quite a few celebrations that are not in the list and that I believe are celebrated here in Central America. Most of them are either blessed or recently canonized as saints. I'll try to compile a list and make suggestions when appropriate 🙌🏼

**Comment by emagnier** [comment]
> you wanted to localise keys of all celebrations used in any Spanish speaking country, but you have missed many

@tukusejssirs you are very demanding 😉! All contributions are most welcome, and everyone does what they can, with the time they have 😉.

By the way you both made a great job to perfect this localisation work. Again, a great thank you!

**Comment by tukusejssirs** [comment]
> @tukusejssirs you are very demanding :wink:! All contributions are most welcome, and everyone does what they can, with the time they have :wink:.
> 
> By the way you both made a great job to perfect this localisation work. Again, a great thank you!

You’re right, @emagnier, I am very demanding. :wink: I wish romcal would be as perfect as possible! :smiley: I’m sorry if it made anyone think that their contribution is not welcomed or even is bad.

And although I do it much less than @emagnier, I too thank you @romcal/romcal-contributors for all your work you’ve done in romcal.

**Comment by CodingAleCR** [comment]
No worries from my side, I appreciate the work you guys in order to make this as great as it is, I specialize in mobile development so if someday you require help to make romcal available in other technologies count me in! 🎉🙌🏼

**Comment by emagnier** [comment]
Thank you for your message, and good to know for your mobile experience! 🙌🏼 I'm also building a mobile app for a monastic community. It also use romcal and other layers for contents and monastic specifics. 😄

**Comment by tukusejssirs** [comment]
@emagnier, I wish you port/implement the code for the monastic/religious calendar into romcal! I don’t say it is the most important thing to do right now, but it would be quite a great addition! :smiley:

**Pull request merged**